### ISSUE TITLE
Calibration upload 71a48a

### ISSUE THREAD

**Issue by bronger** [lensfun/lensfun#174]
Calibration images were uploaded to the directory that starts with “`71a48a_`”.

Please read the [workflow description](https://github.com/lensfun/lensfun/blob/master/tools/calibration_webserver/workflow.rst) for further instructions about the calibration.

Additional comments by the uploader:

> Hi,
> Please, find pictures taken with Nikon AW1 + Nikon AW10 f/2.8 at different apertures.
> Hope it will help adding this lens into lensfun tool.
> Many thanks for your very appreciated work.
> Best regards,
> Yvan Radenac

**Comment by junkyardsparkle** [comment]
The [info from Nikon](http://imaging.nikon.com/lineup/acil/lenses/1_nikkor_aw_10mm_f28/index.htm) made it sound like this might have the same optics as the non-AW version, but the corrections weren't even close. Fancy mustache, for a prime. :)

@bronger: I tried out the --complex-tca switch for this one, with improved results, but the first run simply used the same values as generated without the switch, plus an additional zero parameter. I had to delete the files in the tca directory to get it to re-evaluate. Is this by design, or should the script be patched?
